Add New Vegas Redesigned 3 Revised

Yinsolaya opened the pull request:
Add bash tags

- NPC.Eyes, NPC.Facegen, NPC.Hair, NPC Race tags were added to the main mod and its patches due to extensive edits made to related records.
- Graphics tag was added to NVR3_ADAMTrooper Gloves.Patch due to changes to bipeds. 
- Actors.Factions tag was added to LegionQuestsExpanded Patch.esp due to faction related changes to NPCs.
- Delev, Relev tags were added to NVR3_1nivVSLArmors Patch.esp to ensure that its leveled list changes are carried over.

Yinsolaya committed:
Add New Vegas Redesigned 3 Revised

Add bash tags

NPC.Eyes, NPC.Facegen, NPC.Hair, NPC Race tags were added to the main mod and its patches due to extensive edits made to related records.
Graphics tag was added to NVR3_ADAMTrooper Gloves.Patch due to changes to bipeds.
Actors.Factions tag was added to LegionQuestsExpanded Patch.esp due to faction related changes to NPCs.
Delev, Relev tags were added to NVR3_1nivVSLArmors Patch.esp to ensure that its leveled list changes are carried over.

Yinsolaya committed:
Add New Vegas Redesigned 3 and Replace Original

Add bash tags
- NPC.Eyes, NPC.Facegen, NPC.Hair, NPC Race tags were added to the main mod and its patches due to extensive edits made to related records.
Actors.Factions tag was added to LegionQuestsExpanded Patch.esp due to faction related changes to NPCs.
Delev, Relev tags were added to NVR3_1nivVSLArmors Patch.esp to ensure that its leveled list changes are carried over.

Proposal to replace is due to Revised being the same exact mod for its Legacy version, but cleaned and improved. Also, build checks failed because it couldn't include the same exact esm.

Yinsolaya committed:
Add New Vegas Redesigned Revised, Replace Original

Add bash tags
- NPC.Eyes, NPC.Facegen, NPC.Hair, NPC Race tags were added to the main mod and its patches due to extensive edits made to related records.
Actors.Factions tag was added to LegionQuestsExpanded Patch.esp due to faction related changes to NPCs.
Delev, Relev tags were added to NVR3_1nivVSLArmors Patch.esp to ensure that its leveled list changes are carried over.

Proposal to replace is due to Revised being the same exact mod for its Legacy version, but cleaned and improved. Also, build checks failed because it couldn't include the same exact esm.

Yinsolaya commented:
Sorry, was fiddling around again. I needed to correct one thing from this commit proposal anyways.

Yinsolaya committed:
Add New Vegas Redesigned 3 Revised, Replace OG

Add bash tags
- NPC.Eyes, NPC.Facegen, NPC.Hair, NPC Race tags were added to the main mod and its patches due to extensive edits made to related records.
- Delev, Relev tags were added to NVR3_1nivVSLArmors Patch.esp to ensure that its leveled list changes are carried over.

Proposal to replace is due to Revised being the same exact mod for its Legacy version, but cleaned and improved. Also, build checks failed because it couldn't include the same exact esm.

MacSplody commented on masterlist.txt:
There's an extra file here for some reason?

Yinsolaya replied:
Yeah, once again another mistake. :(
Okay, the latest commit should hopefully be fine though.

MacSplody committed:
Revert "Add New Vegas Redesigned 3 and Replace Original"

This reverts commit f12fdb7acd5af62a6c98bb55358556b2fba32b91.

MacSplody commented on masterlist.yaml:
```diff
@@ -5956,7 +5956,7 @@ plugins:
 
   - name: 'New Vegas Redesigned 3.esm'
-    url: [ 'https://www.nexusmods.com/newvegas/mods/56312/' ]
+    url: [ 'https://www.nexusmods.com/newvegas/mods/72951' ]
```
We can keep the link for the old version if it's still available.
Trailing slashes `/` should be added to links as well, they're useful if you want to change multiple links quickly.

```suggestion
    url:
      - link: 'https://www.nexusmods.com/newvegas/mods/56312/'
        name: 'Fallout New Vegas Redesigned 3 - Open Source'
      - link: 'https://www.nexusmods.com/newvegas/mods/72951/'
        name: 'NVR3R - New Vegas Redesigned 3 Revised'
```

Yinsolaya replied:
Sounds good! Yeah, Open Source is indeed the original.

MacSplody commented on masterlist.yaml:
```diff
@@ -5969,54 +5969,28 @@ plugins:
+      - NPC.Hair
       - NPC.Race
-      - Relev
-    dirty:
-      # v4.4
-      - <<: *quickClean
-        crc: 0xA49C36C8
-        util: '[FNVEdit v4.0.2f](https://www.nexusmods.com/newvegas/mods/34703)'
-        itm: 1001
-        udr: 74
```
I would keep the cleaning data as well, just in case anyone installs the old version.
But I'm flexible on that, if it's pretty well known that this version shouldn't be used.

Does either version provide version info in the plugin file header?
If the new one does, than there could be a case for removing the cleaning data.
As we could inform them of the newer version.

Yinsolaya replied:
I'll check rn. And good idea.

Yinsolaya replied:
Okay, so both versions on the Header say "1.340000". That is what you're asking for, right?

MacSplody replied:
> Okay, so both versions on the Header say "1.340000". That is what you're asking for, right?

It should be under `SNAM - Description`

![image](https://github.com/loot/falloutnv/assets/1944639/24fcf126-01be-4576-83c7-1555a4e306bb)

Yinsolaya replied:
In that case, no. Nothing in SNAM.

MacSplody replied:
Well you could check the new version files and see if they are clean.
If they are clean then the CRC is reliable, and we could use that for an update message.
And then there would be no need to keep the cleaning data for the old version.

Yinsolaya replied:
Fair fair! I shall do it!

Yinsolaya replied:
clean:
      - crc: 0x972A47BB
        util: 'FNVEdit v4.1.5 EXPERIMENTAL'

As expected, it is indeed clean.

MacSplody committed:
Update location

- Add mod names.
- Add link to latest version.
- https://www.nexusmods.com/newvegas/mods/72951/

MacSplody committed:
Update location

- Add another location.
- Qwinn's Refined FNV Redesigned 3.
- https://www.nexusmods.com/newvegas/mods/65865/

MacSplody committed:
Format

MacSplody committed:
Add cleaning data

- Version 1.4.4 Legacy
- Version 2.1.0

MacSplody committed:
Format

- Add sub sections.

MacSplody committed:
Add message

- Old version esp file is obsolete.

MacSplody committed:
Format

- Rearrange files by name.

MacSplody committed:
Remove links

MacSplody committed:
Correction

- Different results with vanilla masters.
- TTW masters were used previously.

MacSplody committed:
format

- Tag order.

MacSplody committed:
Add comments

- For patched mods.

MacSplody commented:
Thanks.
